ratelimits/transaction.go

```diff
@@ -241,15 +253,15 @@ func (builder *TransactionBuilder) FailedAuthorizationsPerDomainPerAccountCheckO
 	}
 
 	var txns []Transaction
-	for _, name := range orderDomains {
+	for _, ident := range orderIdents {
 		// FailedAuthorizationsPerDomainPerAccount limit uses the
-		// 'enum:regId:domain' bucket key format for transactions.
-		perDomainPerAccountBucketKey := NewRegIdDomainBucketKey(FailedAuthorizationsPerDomainPerAccount, regId, name)
+		// 'enum:regId:identValue' bucket key format for transactions.
+		perIdentValuePerAccountBucketKey := NewRegIdIdentValueBucketKey(FailedAuthorizationsPerDomainPerAccount, regId, ident)
 
-		// Add a check-only transaction for each per domain per account bucket.
-		// The cost is 0, as we are only checking that the account and domain
-		// pair aren't already over the limit.
-		txn, err := newCheckOnlyTransaction(limit, perDomainPerAccountBucketKey, 1)
+		// Add a check-only transaction for each per identValue per account
+		// bucket. The cost is 0, as we are only checking that the account and
+		// identValue pair aren't already over the limit.
+		txn, err := newCheckOnlyTransaction(limit, perIdentValuePerAccountBucketKey, 1)
```

Comment:
This is a pre-existing problem that I noticed because of the comment diff: it says the cost is 0, but the `cost` parameter is 1. I think that part of the comment is out-of-date and should be deleted.